crates/sui-move-build/src/linters/mod.rs

```diff
@@ -7,23 +7,39 @@ use move_compiler::{
 };
 use move_ir_types::location::Loc;
 
+pub mod coin_field;
 pub mod custom_state_change;
 pub mod self_transfer;
 pub mod share_owned;
 
+pub const SUI_PKG_NAME: &str = "sui";
+
+pub const TRANSFER_MOD_NAME: &str = "transfer";
+pub const TRANSFER_FUN: &str = "transfer";
+pub const PUBLIC_TRANSFER_FUN: &str = "public_transfer";
+pub const SHARE_FUN: &str = "share_object";
+pub const PUBLIC_SHARE_FUN: &str = "public_share_object";
+pub const FREEZE_FUN: &str = "freeze_object";
+
+pub const COIN_MOD_NAME: &str = "coin";
+pub const COIN_STRUCT_NAME: &str = "Coin";
+
 pub const SHARE_OWNED_DIAG_CATEGORY: u8 = 1;
 pub const SHARE_OWNED_DIAG_CODE: u8 = 1;
 pub const SELF_TRANSFER_DIAG_CATEGORY: u8 = 2;
 pub const SELF_TRANSFER_DIAG_CODE: u8 = 1;
 pub const CUSTOM_STATE_CHANGE_DIAG_CATEGORY: u8 = 3;
 pub const CUSTOM_STATE_CHANGE_DIAG_CODE: u8 = 1;
+pub const COIN_FIELD_DIAG_CATEGORY: u8 = 4;
```

Comment:
Yeah, it's probably time to change it to an enum. The idea is that categories should be unique and codes within a category as well, but at this point we only have one code per category and I don't think this will change in the near future. Modified the code accordingly.